client/securedrop_client/gui/widgets.py

```diff
@@ -2860,9 +2868,13 @@ def add_file(self, file: File, index: int) -> None:
         """
         Add a file from the source.
         """
+        # A refactor now passes the File object directly into the FileWidget constructor,
```

Comment:
Mostly a general note, I don't think describing the history is necessary, we will find that context in the git history. Just the second-half of the comment is needed.